sites/public/pages/finder.tsx

```diff
@@ -133,7 +140,38 @@ const Finder = () => {
           formSection: t("finder.progress.housingLabel"),
           fieldGroupName: "rentalCosts",
           fields: costFields,
+          question: t("finder.rentalCosts.question"),
+          subtitle: t("finder.default.subtitle"),
         })
+
+        const a11yFields = listingFeatures.map((elem) => ({
```

Comment:
When running this locally several of the listing features don't have translation keys. Is that expected? 
![image](https://user-images.githubusercontent.com/42942267/196464884-b31a3683-deca-4848-967c-0c1a82951c6d.png)

Reply:
@ludtkemorgan I was only able to reproduce this by running yarn db:reseed in backend/core rather than yarn db:reseed:detroit since the programs are pulled from the backend. Let me know if that fixes things on your side!

Reply:
Yeah, this was not an issue, just my local seed was incorrect